river/Cursor.zig

```diff
@@ -165,10 +176,17 @@ pub fn init(self: *Self, seat: *Seat) !void {
 }
 
 pub fn deinit(self: *Self) void {
+    self.hide_cursor_timer.remove();
     self.xcursor_manager.destroy();
     self.wlr_cursor.destroy();
 }
 
+fn getHideTimeout(self: *Self) u31 {
+    if (self.pressed_count > 0)
+        return 0;
```

Comment:
Style nit: I prefer either
```zig
if (self.pressed_count > 0) return 0;
```
or
```zig
if (self.pressed_count > 0) {
    return 0;
}
```
I'm sure you've seen some existing old examples of this style around the codebase, but I've been consistently updating them as I touch that code. I should really get around to codifying this in CONTRIBUTING.md.